src/components/commons/InputComponent.tsx

```diff
@@ -79,6 +79,7 @@ const InputComponent = forwardRef<HTMLInputElement, InputProps>(
             />
           )}
         </div>
+        {error && isVisited && <div className="text-red-800 mt-2">{error}</div>}
```

Comment:
Sorry @fullstackninja864, can you also check the other components thats already using this one?

_Go to any verified contract -> Contravt -> Write contract -> Add zeroes to any uint256 input field_
<img width="316" alt="image" src="https://github.com/BirthdayResearch/metascan/assets/17063281/26f2c775-42ad-4530-bc90-306c1cdc4a7e">

Reply:
Can fix this in another PR